tests/pytorch/test_pytorch_model_export.py

```diff
@@ -907,6 +907,25 @@ def test_load_model_raises_exception_when_pickle_module_cannot_be_imported(
     assert bad_pickle_module_name in str(exc_info)
 
 
+@pytest.mark.large
+def test_pyfunc_serve_and_score(data):
```

Comment:
`tests/pytorch/test_pytorch_model_export.py` contains a few serving tests but they all require `--no-conda` regardless of torch version so I created this test.